Makefile.toml

```diff
@@ -463,6 +464,26 @@ fi
 '''
 ]
 
+[tasks.check-golangci-lint]
+script = [
+'''
+top_path=$(pwd)
+config_path=$top_path/.golangci.yaml
```

Comment:
Nit: to be more consistent with the variable expansion in the rest of the file:
```
config_path="${top_path}/.golangci.yaml"
```
There's a few uses below as well. :)

Reply:
True - these paths should be wrapped to avoid unintentional escape

---

One note however: we have vastly inconsistent escape patterns in our Makefile shell code. So I just went with what was easiest.

This uses the `{}` to escape the path:
https://github.com/bottlerocket-os/bottlerocket/blob/f825677d128e0931b57efe08dfc0424968e055fc/Makefile.toml#L414

This only wraps the statement in a string:
 https://github.com/bottlerocket-os/bottlerocket/blob/f825677d128e0931b57efe08dfc0424968e055fc/Makefile.toml#L336

This wraps in single quotes `''` (although there's no risk of unescaped path causing problems here):
https://github.com/bottlerocket-os/bottlerocket/blob/f825677d128e0931b57efe08dfc0424968e055fc/Makefile.toml#L527

We should consider a way to shell check the Makefile targets. Although, that may be weird / challenging since it's only a toml file